.github/workflows/gitpod.yml

```diff
@@ -0,0 +1,56 @@
+name: Build Gitpod Docker image
+
+on:
+  push:
+    branches:
+      - feature/**
+  schedule:
+    - cron: "0 0 * * *" # nightly
```

Comment:
Why does this have both an `on: push:` and `on: schedule:`? Seems like we build on every PR merge to `main`, there's no need to build nightly?

Reply:
I added the `on: push` for debugging. And was planning to leave the build as `nightly` as we did for SciPy or would you prefer having it on `push` instead?